src/viewer/index.ts

```diff
@@ -215,6 +217,10 @@ class Viewer {
       return;
     }
 
+    if (this.frameCount % this.raycastFrameInterval !== 0) {
+      return;
+    }
```

Comment:
I think it is safe to assume that normal people will turn off the browser before ```frameCount``` hits max integer size, but I have traditionally seen this as:

```js
if (this.frameCount >= this.raycastFrameInterval) {
    this.frameCount = 0;
} else {
    return;
}
 ```
 
 For this application I don't think it matters, but I thought I would point it out since it looked odd.

Reply:
I thought about it and then promptly forgot. I should probably do this.